playwright-e2e/page-objects/local-authority/birth-father/lsBirthFatherLastKnownAddressPage.page.ts

```diff
@@ -0,0 +1,40 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class LABirthFatherLastKnownAddressPage extends BasePage {
```

Comment:
seems to be a typo in the file name that isn't reflected in the class name?
"ls" not "la"